skim/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2018"
 [lib]
 name = "skim"
 path = "src/lib.rs"
+doctest = false
```

Comment:
would be nice to have doctest in the future

Reply:
For now, I disabled it since we need indented blocks for the man page formatting, but these are ran as doctests by default. Right now, there is only one doctest in the crate afaik, so we don't lose too much.